### PR TITLE
US#479021 Control replicas in helm charts

### PR DESCRIPTION
**Pull request by hasan-mahmood-opentext**
- Updated idol-library `_deployment.tpl` to get number of replicas from relevant values.yaml.
 - Updated all relevant charts to use default number of replicas (1).

### CODE REVIEW

**Review by daniel-lau-mf — COMMENTED** [review]
Please update idol-library-example-test as well.

**Review by daniel-lau-mf — CHANGES_REQUESTED** [review]
Mostly good - see feedback comments for some extra things to do.

File: helm/distributed-idol/values.yaml [hunk 1]
```diff
@@ -272,6 +272,9 @@ dah:
   # to provide required TLS certificates
   usingTLS: false
 
+  # replicas -- number of replica pods for this container (defaults to 1)
```
[the remark this answers is not in the document]
Comment (hasan-mahmood-opentext) [reply helm/distributed-idol/values.yaml:275]: Is this only for this case or for the other charts as well?

[the remark this answers is not in the document]
Comment (daniel-lau-mf) [reply helm/distributed-idol/values.yaml:275]: Just this case.

File: helm/release/distributed-idol-0.9.0.tgz
[the remark this answers is not in the document]
Comment (hasan-mahmood-opentext) [reply helm/release/distributed-idol-0.9.0.tgz:1]: yep, updated

**Review by daniel-lau-mf — APPROVED** [review]

**Pull request merged**